dev-aws/kafka-shared-msk/energy-billing/modules.tf

```diff
@@ -6,15 +6,15 @@ module "billing_adapter" {
 
 module "billing_adapter_retry_1" {
   source           = "../../../modules/tls-app"
-  cert_common_name = "energy-billing/billing-adapter-retry-1"
+  cert_common_name = "energy-billing/billing-adapter"
```

Comment:
Linting is failing. See https://github.com/utilitywarehouse/tflint-ruleset-kafka-config/blob/main/rules/msk_unique_app_names.md

Reply:
To run linters locally, please see: https://github.com/utilitywarehouse/kafka-cluster-config/blob/main/CONTRIBUTING.md#linting